app/controllers/v1/terms_controller.rb

```diff
@@ -96,11 +97,24 @@ def options
 
     private
 
+      def page
+        page = (params[:page].blank?) ? 1 : params[:page].to_i
+        page = 1 if page < 1
+        page
+      end
+
+      def per_page
+        per_page = (params[:per_page].blank?) ? URIService::PER_PAGE_DEFAULT : params[:per_page].to_i
+        per_page = URIService::PER_PAGE_DEFAULT if per_page < 1
```

Comment:
We may want to put a limit on the maximum number per page, just so someone doesn't ever accidentally (or intentionally) ask for 100,000 terms per page.  Maybe max out at 1000 per page?

Reply:
I agree, that's a good feature. I didn't know if we wanted to leave that up to the users since they are authenticated. What would be a good max number of terms per page?

Reply:
Would 1000 be too low, or does that seem okay?

Reply:
I think that seems okay. Do you see yourself ever wanting more than 1000 terms per request?

Reply:
Not for a single request, no.  If I'm ever iterating through **all** terms in a vocabulary for some reason, I'm fine with just doing multiple paged requests.  Now that I think about it, I wonder if 1000 is too high.  If there's a lot of activity hitting URI Service, is returning 1000 results from solr a lot?  (I don't actually know.)  Can I revise my suggestion to 200 max, and we'll increase it later if that ends up being too restrictive?

Reply:
Done.